src/js/components/Calendar/utils.js

```diff
@@ -150,7 +158,17 @@ export const normalizeForTimezone = (value, timestamp, normalize = true) => {
   let localOffset = 0;
   if (normalize) {
     if (timestamp && typeof timestamp === 'string') {
-      hourDelta = parseInt(timestamp?.split(':')[0], 10);
+      const day = Array.isArray(value) ? undefined : new Date(value);
+      const today = new Date();
+      if (
+        day &&
+        !inDaylightSavings(day) &&
```

Comment:
What about the other direction, where `day` is inDaylightSavings but `today` isn't?

Reply:
Do we have to check whether `inDaylightSavings(day) !== inDaylightSavings(today)`?